backend/util/environment.go

```diff
@@ -14,7 +14,7 @@ var (
 )
 
 func IsHubspotEnabled() bool {
-	return !IsDevEnv()
+	return !IsDevEnv() && !IsTestEnv()
```

Comment:
Was getting a very tricky panic to debug: https://github.com/highlight/highlight/actions/runs/5225688025/jobs/9435696575

Turns out it was because Hubspot was getting called here:

https://github.com/highlight/highlight/blob/c7c5b2c63d51228b33c917c70b736b8d8bf7e3bd/backend/public-graph/graph/resolver.go#L1128-L1129